subworkflows/nf-core/vcf_extract_relate_somalier/meta.yml

```
@@ -39,8 +41,10 @@ input:
   - ch_peds:
       type: file
       description: |
-        Optional - A channel containing the PED files for the corresponding families
-        Structure: [ meta, ped ]
+        A channel containing an optional PED file for the corresponding families.
```

```suggestion
        A channel containing an mandatory PED file for the corresponding families.
```

According to the docs-line in the module?

---

It can be `[]` so it's not mandatory, I'll make that more clear in the module :)